packages/shared/create-local-vue.js

```diff
@@ -33,7 +33,7 @@ function _createLocalVue(
   })
 
   // config is not enumerable
-  instance.config = cloneDeep(Vue.config)
+  instance.config = merge({}, cloneDeep(Vue.config), config)
```

Comment:
Could you add a test to assert / document this feature? 🙌

Reply:
I don't even know if we need to use `merge` here - can't we just do:

```
instance.config = {...Vue.config, ...config)
```

Or something?


```

Reply:
@lmiller1990 it depends! we can do that but `Lodash` merge is faster than spread operator and use less memory. I don't mind to use any of them, let me know what do you prefer

more info: https://www.measurethat.net/Benchmarks/Show/2978/0/lodash-merge-vs-objectassign-vs-spread